axiom/monitors.go

```diff
@@ -106,40 +106,47 @@ func (c *MonitorType) UnmarshalJSON(b []byte) (err error) {
 }
 
 type Monitor struct {
+	// Identification
```

Comment:
```suggestion
	// Identification

```

Otherwise it looks odd in GoDoc.